### PR TITLE
Support multiple campuses

### PR DESCRIPTION
**Pull request by therealsujitk**
I've added JSON schema's in [`/src/schemas/`](https://github.com/therealsujitk/FFCSonTheGo/tree/add-timetable-schema/src/schemas) for each campus that can be changed easily in future.

To test slot clashing you can use the courses **CSE1001 - L35+L36+L37+L38** and **MAT1014 - B2+TB2+TBB2+V4**.

FFCS On The Go Default Table (Timetable)

FFCS On The Go Default Table (Course List)

Let me know if you find any issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
         "bootstrap": "^5.0.2",
         "bootstrap-select": "^1.13.18",
         "bootstrap4": "npm:bootstrap@4.6.0",
+        "datejs": "^1.0.0-rc3",
```
Comment (vatz88) [inline package.json:44]: Instead of datejs, i'd recommend using https://date-fns.org/ which is more popular these days and is well maintained.

Comment (therealsujitk) [reply package.json:44]: Alright, will change it as soon as I get some time.

**Comment by therealsujitk** [comment]
@vatz88 I've migrated to [`date-fns`](https://date-fns.org/), but I'm leaving this PR as a draft for now because I'm not really sure how users expect Quick Visualization to work. I'll have to look at the production code in [`master`](https://github.com/vatz88/FFCSonTheGo/tree/master) to see how it currently works and if I can improve it somehow.

**Comment by vatz88** [comment]
Looks good to me. It's lot of code changes, haven't reviewed it all. Hope you have tested it well. Approving, feel free to merge it when you feel confident

**Review by vatz88 — APPROVED** [review]

**Pull request merged**